### DISCUSSION TITLE
I try to install espeak -ng -1.5.1 in ubuntu 18.04 but I got this error :[dictionary.h: No such file or directory]

### DISCUSSION THREAD

**Discussion by Envelopepiano** [espeak-ng/espeak-ng#1397] [Q&A]
Hi everyone, I'm a newbie to tts. Recently, I tried to install the new espeak for Ukrainian, I downloaded the 1.5.1 file to linux and unzipped it. After cd to the directory and ./configer, it was normal, but after  the make command, this problem appeared : 
**

> fatal error: dictionary.h: No such file or directory.

**
![image](https://user-images.githubusercontent.com/101227613/192104403-3be86751-734e-495c-bb38-fd48992dc65b.png)

 seems to be missing some file, can anyone help me? Thanks!!

**Comment by jaacoppi** [comment]
See docs/building.md for build instructions. Did you run ./autogen.sh before ./configure?

What file did you download and where? Verify that dictionary.h has been unzipped. You can see where it should be by comparing to the directory structure of this git repo.

**Reply by Envelopepiano** [reply]
Thank you very much sir! I found that I've downloaded the wrong files , I appreciate for your help!

**Reply by sunjianan9900** [reply]
Hi, I have the same problem.

`espeak-ng-1.51.tar.gz` has no file . /autogen.sh
`Source code(tar.gz)` has no file  . /configure

How and where should I execute both
 . /autogen.sh
 . /configure
and then make?

**Reply by sunjianan9900** [reply]
Could you please tell me which file I should download (linux) and then which commands to execute? Because I can't execute all commands smoothly in both downloaded files

**Reply by alex19EP** [reply]
try https://github.com/espeak-ng/espeak-ng/archive/refs/tags/1.51.tar.gz

**Reply by alex19EP** [reply]
> Source code(tar.gz) has no file . /configure

you need to run ./autogen.sh first

**Reply by sunjianan9900** [reply]
> > Source code(tar.gz) has no file . /configure
> 
> you need to run ./autogen.sh first

Thanks, it worked!
BTW, I finished compiling the file on Linux.
Is it possible to put it into apline linux and run it?
I don't want to compile on apline linux because I don't want to increase its size.
